docs/guide-zh-CN/input-validation.md

```diff
@@ -334,7 +334,7 @@ class CountryValidator extends Validator
     public function validateAttribute($model, $attribute)
     {
         if (!in_array($model->$attribute, ['兲朝', '墙外'])) {
-            $this->addError($attribute, '国家必须为 "兲朝" 或 "墙外" 中的一个。');
+            $model->addError($attribute, '国家必须为 "兲朝" 或 "墙外" 中的一个。');
```

Comment:
Well, it is in the English text. Translation is, obviously, wrong. If you can update it according to English docs it would be great.